apps/watchdog/src/deploy_with_compose.rs

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) SimpleStaking and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+#![forbid(unsafe_code)]
+
+use std::path::PathBuf;
+use std::process::{Command, Output};
+
+use slog::{info, Logger};
+use tokio::time::{sleep, Duration};
+
+use crate::image::{Explorer, OcamlDebugger, Sandbox, TezedgeDebugger, WatchdogContainer};
+use crate::node::{OcamlNode, TezedgeNode};
+
+// TODO: use external docker-compose for now, should we manage the images/containers directly?
+pub async fn launch_stack(compose_file_path: &PathBuf, log: &Logger) {
+    start_with_compose(compose_file_path, Explorer::NAME, "explorer");
+    start_with_compose(compose_file_path, TezedgeDebugger::NAME, "tezedge-debugger");
+    // debugger healthcheck
+    while reqwest::get("http://localhost:17732/v2/log").await.is_err() {
```

Comment:
@adonagy 
please, create const for `http://localhost:17732/v2/log` (maybe it should be better as part of configuration, but at least const is better)

Reply:
@adonagy 
I see below there is also a constant:
`pub const TEZEDGE_DEBUGGER_PORT: u16 = 17732;`
I would put that to one place, maybe just struct with constants